test/source/tests/settings.ts

```diff
@@ -285,7 +285,13 @@ export let defineSettingsTests = (testVariant: TestVariant, testWithBrowser: Tes
       const myKeyFrame = await SettingsPageRecipe.awaitNewPageFrame(settingsPage, `@action-show-key-1`, ['my_key.htm', 'placement=settings']);
       await Util.sleep(1);
       await myKeyFrame.waitAll('@content-fingerprint');
+      await myKeyFrame.notPresent('@action-update-prv');
```

Comment:
we should rename the test 'settings - my key page - remove button should be hidden when using key manager' too